app/assets/stylesheets/mixins/_forum_post.scss

```diff
@@ -0,0 +1,38 @@
+@mixin forum_post {
```

Comment:
Name of mixin `forum_post` should be written in all lowercase letters with hyphens instead of underscores